include/msgpack/adaptor/cpp11/unordered_map.hpp

```diff
@@ -42,9 +42,9 @@ struct convert<std::unordered_map<K, V>> {
         for(; p != pend; ++p) {
             K key;
             p->key.convert(key);
-            p->val.convert(tmp[key]);
+            p->val.convert(tmp[std::move(key)]);
         }
-        tmp.swap(v);
+        v = std::move(v);
```

Comment:
Some of tests are failed. See https://travis-ci.org/msgpack/msgpack-c/jobs/62302916
It seems that
```C++
v = std::move(v)
```
should be
```C++
v = std::move(tmp)
```